lib/fauxpaas/artifact_builder.rb

```diff
@@ -38,5 +39,12 @@ def add_reference(ref, path)
         .write
     end
 
+    def install_local_gems(path)
+      raise "bundler failed to install gems in #{path}" unless Dir.chdir(path) do
```

Comment:
Hmm... I'm not sure how I feel about the guard clause style for a detailed operation, especially with a block. This one is pretty tidy, so no request for change here -- just a style note for consideration.